src/vos/vos_common.c

```diff
@@ -724,11 +724,6 @@ vos_self_nvme_init(const char *vos_path, uint32_t tgt_id)
 		goto out;
 
 	self_mode.self_nvme_init = true;
-	rc = bio_xsctxt_alloc(&self_mode.self_xs_ctxt, tgt_id, true);
-	if (rc) {
-		D_ERROR("Failed to allocate NVMe context. "DF_RC"\n", DP_RC(rc));
-		goto out;
-	}
```

Comment:
This his removed from vos_self_nvme_init() but the vos_self_nvme_fini() isn't updated accordingly?